bunfig.toml

```diff
@@ -1,5 +1,2 @@
-[install]
-exact = true
-
```

Comment:
I added the exact versioning enforcement to `.npmrc`. We'll need to keep `bunfig.toml` around for now for testing.